src/libraries/Microsoft.Extensions.Logging/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,402 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
```

Comment:
NIT: When adding baseline files, it is nice to add a comment that explains why suppressions are being added so people don't need to find the PR that added them to figure out why they were needed.
```suggestion
  <!-- The below breaking changes are all due to new nullable annotations that are being added in this release -->
  <Suppression>
```